src/Tools/ExternalAccess/Razor/RazorUri.cs

```diff
@@ -4,14 +4,19 @@
 
 using System;
 using Microsoft.CodeAnalysis.LanguageServer;
+using Roslyn.LanguageServer.Protocol;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.Razor;
 
 internal static class RazorUri
 {
+    [Obsolete("Use RazorUri.GetUriFromFilePath instead")]
```

Comment:
Nit: Shouldn't this tell people to use `CreateAbsoluteDocumentUri`?

Although, if changing to DocumentUri is a breaking change for Razor, is there a reason to keep this obsolete at all?

Reply:
So changing to documenturi doesn't seem to break non-cohosted razor.  I'm not as sure about this one